yamllint/rules/truthy.py

```diff
@@ -63,6 +71,25 @@
     yes:  1
     on:   2
     True: 3
+
+#. With ``truthy: {allowed-values: ["yes", "no"]}``
+
+   the following code snippet would **PASS**:
+   ::
+
+    key1: yes
+    key2: no
+    key3: foo
+    key4: bar
+
+   the following code snippet would **FAIL**:
+   ::
+
+    key1: no
+    key2: true
+    key3: false
+    key4: yes
+    key5: foo
```

Comment:
For clarity, I propose:
```
   the following code snippet would **PASS**:
   ::

    - yes
    - no
    - "true"
    - 'false'
    - foo
    - bar

   the following code snippet would **FAIL**:
   ::

    - true
    - false
    - on
    - off
```
What do you think?

Reply:
Yes, it is easier to understand.